cmd/chaos/chaos.go

```diff
@@ -1,8 +1,139 @@
 package main
 
-import "github.com/projectdiscovery/chaos-client/internal/runner"
+import (
+	"fmt"
+	"os"
+
+	"github.com/alecthomas/kong"
+	"github.com/projectdiscovery/chaos-client/internal"
+	"github.com/projectdiscovery/chaos-client/internal/dns"
+	"github.com/projectdiscovery/chaos-client/internal/subdomains"
+	"github.com/projectdiscovery/gologger"
+	"github.com/projectdiscovery/gologger/levels"
+	updateutils "github.com/projectdiscovery/utils/update"
+)
+
+var cli struct {
+	Key                string `help:"Chaos key for API" short:"k"`
+	Silent             bool   `help:"Make the output silent"`
+	Output             string `help:"File to write output to (optional)" short:"o"`
+	Version            bool   `help:"Show version of chaos"`
+	Verbose            bool   `help:"Verbose" short:"v"`
+	Update             bool   `help:"update Chaos to latest version" aliases:"up"`
+	DisableUpdateCheck bool   `help:"disable automatic Chaos update check" aliases:"duc"`
+	Subdomains         struct {
+		Domain string `arg:"" help:"Domain to search for subdomains"`
+		Count  bool   `help:"Show statistics for the specified domain"`
+		JSON   bool   `help:"Print output as json"`
+	} `cmd:"" help:"List subdomains"`
+	SubdomainsBatch struct {
```

Comment:
Optional: Adding a sub-command specifically for the file could unnecessarily complicate a simple matter. How about we move it under `Subdomains`?

Reply:
Ahh, the thing is `subdomains` subcommand received domain: `chaos subdomains <domain>` I was thinking about this and it looks like we can't omit domain and do something like `chaos subdomains -f file.txt` So I decided to move it to separate subcommand. In theory we can use something like `chaos subdomains <file>` but how chaos client will know whether filename is filename or domain name :D  We can also do this: `chaos subdomains <file> -f` where `-f` flag indicates that argument is file.